web/index.html

```diff
@@ -18,7 +18,7 @@
     <title>Yote</title>
   </head>
   <body class="bg-gray-50">
-    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <noscript>You need to enable JavaScript to run this app ROOT ONE.</noscript>
```

Comment:
?

Reply:
oh i thought i killed this, testing stuff. i'll remove.